e2e/cypress/integration/polis/client-participation/embeds.spec.js

```diff
@@ -0,0 +1,156 @@
+describe('Embedded Conversations', () => {
+  // This test requires overriding client-admin/embed.html with
+  // e2e/cypress/fixtures/html/embeds.html
+  const POLIS_DOMAIN = Cypress.config().baseUrl.replace('https://', '')
+  const CONVO_DESCRIPTION = 'This is dummy description for embed tests.'
+  const CONVO_TOPIC = 'Embed test topic'
+
+  beforeEach(function () {
+    cy.createConvo('moderator').then(() => {
+      cy.seedComment('Seed statement 1', this.convoId)
+      cy.get('[data-test-id="description"]').type(CONVO_DESCRIPTION).blur()
+      cy.get('[data-test-id="topic"]').type(CONVO_TOPIC).blur()
+    })
+  })
+
+  it('renders a default embed', function () {
+    cy.logout()
+    cy.visit(`${Cypress.config().baseUrl}/embed.html?polisDomain=${POLIS_DOMAIN}&data-conversation_id=${this.convoId}`)
+    cy.enter(`#polis_${this.convoId}`).then(cyframe => {
+      cyframe().find('div[data-view-name="root"]').as('iframe')
```

Comment:
ah, so this is how you test iframes with Cypress. nifty. https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/